xml/rmt_mirroring.xml

```diff
@@ -57,7 +58,7 @@
    </step>
    <step>
     <para>
-     Click <guimenu>Organization</guimenu> in the top menu.
+     Select <guimenu>Organization</guimenu> in the top menu.
```

Comment:
There's a new `Proxies` tab now, which has the details about SMT/RMT/SUMA instances -- the credentials are also there. I think at some point the credentials will be removed from `Organization` tab.